tapglue-android-sdk/src/main/java/com/tapglue/networking/TGCustomCacheObject.java

```diff
@@ -45,9 +52,10 @@
      * Type of cache object - to be used to determine type of generic API object inside requests
      */
     public enum TGCacheObjectType {
-        Connection(1), ConnectionUser(2), ConnectionUserList(3), Event(4), EventObject(5), Feed(6),
+        Connection(1), ConnectionUser(2), ConnectionUserList(3), Event(4), EventObject(5), EVENTSLIST(6),
```

Comment:
Please rename `EVENTSLIST` to `EventList` to be consistent with all the other declarations.